packages/core-components/src/components/SimpleStepper/types.ts

```diff
@@ -31,6 +31,11 @@ export type StepActions = {
   canRestart?: () => boolean;
   onRestart?: () => void;
   restartText?: string;
+
+  showSkip?: boolean;
+  canSkip?: () => boolean;
```

Comment:
Just gotta point out that this prop is really awkward, but w/e, better have it fit with the rest and potentially refactor in the future